File: bmi.f90
program bmi
        implicit none
        real :: height, weight

        print *, 'Weight, please! (in kg)'
        read (*,*) weight

        print *, 'Now, your height (in m)'
        read(*,*) height

        write(*,'(A, F10.2)') 'Your bmi is ',  (weight / (height**2))
end program bmi

File: bmi.java
import java.io.*;

public class bmi
{
	public static void main(String[] args)
	{
		try
		{
			InputStreamReader isr = new InputStreamReader(System.in);
			BufferedReader br = new BufferedReader(isr);

			System.out.print("Your weight please (in kg please): ");
			float weight = br.readLine();

			System.out.print("Your height now please (in m!): ");
			float height = br.readLine();

			float bmi = weight / Math.pow(height, 2);
			System.out.print(bmi);
		}
		catch (IOException ioe)
		{
		}
	}
}
